.github/workflows/packaging.yml

```diff
@@ -81,6 +81,6 @@ jobs:
             CURL_CMD+=" -F $(basename ${f})=@./build/${f}"
           done
 
-          echo ${CURL_CMD}
+          echo "Upload packages..."
```

Comment:
I just checked: the secret is not actually shown in the logs:

> curl -LfsS -X PUT https://rws.tarantool.org/tarantool-modules/debian/stretch -u *** -F product=tarantool-queue -F tarantool-queue-1.2.2.0.tar.xz=@./build/tarantool-queue-1.2.2.0.tar.xz -F tarantool-queue_1.2.2.0-1.debian.tar.xz=@./build/tarantool-queue_1.2.2.0-1.debian.tar.xz -F tarantool-queue_1.2.2.0-1.dsc=@./build/tarantool-queue_1.2.2.0-1.dsc -F tarantool-queue_1.2.2.0-1_all.deb=@./build/tarantool-queue_1.2.2.0-1_all.deb -F tarantool-queue_1.2.2.0.orig.tar.xz=@./build/tarantool-queue_1.2.2.0.orig.tar.xz

The commit title/message may be confusing.

Reply:
The commit message has been updated.